decamConfig.py

```diff
@@ -2,9 +2,5 @@
 from lsst.obs.decam.decamNullIsr import DecamNullIsrTask
 config.isr.retarget(DecamNullIsrTask)
 
-from lsst.pipe.tasks.setConfigFromEups import setPhotocalConfigFromEups
-from lsst.meas.extensions.astrometryNet import LoadAstrometryNetObjectsTask
-config.calibrate.astromRefObjLoader.retarget(LoadAstrometryNetObjectsTask)
-config.calibrate.photoRefObjLoader.retarget(LoadAstrometryNetObjectsTask)
-
-setPhotocalConfigFromEups(config.calibrate.photoCal)
+from lsst.meas.astrom.matchPessimisticB import MatchPessimisticBTask
+config.calibrate.astrometry.matcher.retarget(MatchPessimisticBTask)
```

Comment:
Don't we still need to retarget the photometric matcher, or does it use the same one?

Reply:
`obs_decam/config/processCcd.py` already defaults to the HTM catalogs for both the astrometric and photometric calibration.

It's actually done this since 2017-07-20.  I was previously explicitly overriding that in the `validation_data_decam/decamConfig.py` to use the astrometry.net files.

Reply:
These new lines are necessary because `obs_decam` doesn't explicitly set `MatchPessimisticBTask`.  Once DM-14868 is merged, it's likely that `MatchPessimisticBTask` will become the `processCcd` config default and I'll remove the lines above.  I'd like to eventually remove any config customizations in the CI runs.